src/App.vue

```diff
@@ -900,19 +903,13 @@ export default class App extends Mixins(
     }
   }
 
-  /** Called when $route property changes. Used to init app. */
-  @Watch('$route', { immediate: true })
+  /** Called when $route property changes. */
+  @Watch('$route', { immediate: false })
   private async onRouteChanged (): Promise<void> {
     // init only if we are not on signin or signout route
     if (!this.isRouteName(RouteNames.SIGN_IN) && !this.isRouteName(RouteNames.SIGN_OUT)) {
       this.assignIdentifier()
-
       this.setCurrentStep(this.$route.meta?.step || 1)
-
-      // wait a moment for token to be available in session storage
-      // and JS date to be fetched from server
-      await Vue.nextTick()
-
       await this.fetchData(true)
```

Comment:
What happened here is the route change handler ran before created(), so the initial time/date was not set.

Now, this function is not run "immediately", and I call it at the end of created() instead. Tested and works fine.